README.md

```diff
@@ -8,7 +8,7 @@
     <img alt="GitHub" src="https://img.shields.io/github/license/stackotter/swift-cross-ui">
 </p>
 
-A SwiftUI-like framework for creating cross-platform apps in Swift. It uses [SwiftGtk](https://github.com/stackotter/SwiftGtk) as its backend (GTK 4.10+).
+A SwiftUI-like framework for creating cross-platform apps in Swift. It uses Gtk 4.10+ as its backend.
```

Comment:
Good catch! But we support any Gtk4 not just 4.10+

Reply:
Fixed 👍